Packs/XMCO/Integrations/Serenety/Serenety.py

```diff
@@ -0,0 +1,254 @@
+
+    def _post(self, url_suffix: str, params: dict[str, Any] | None = None):
+        return self._http_request(
+            method="POST", url_suffix=url_suffix, data=json.dumps(params)
+        )
+
+    def fetch_current_user(self) -> dict[str, Any]:
+        return self._get('/user/current')
+
+    def fetch_alert(self, scope_id: str = None, dry_run: bool = None) -> list[dict[str, Any]]:
+        has_next = True
+        params: dict[str, Any] = {
+            "page": 1
+        }
+        if scope_id:
+            params['scope'] = scope_id
+
+        if dry_run is not None:
+            params['to_sync'] = dry_run
+
+        res = self._get(
+            url_suffix='/ticketor/serenety',
+            params=params
+        )
+        data = res.get('items')
+
+        while has_next:
+            if res and res['page'] < res['pages']:
+                params['page'] += 1
+                res = self._get(
+                    url_suffix='/ticketor/serenety',
+                    params=params
+                )
+
+                data += res.get('items', [])
+            else:
+                has_next = False
+
+        return data
+
+
+''' HELPER FUNCTIONS'''
+
+
+def convert_to_demisto_severity(severity: str) -> float:
+    """
+    Maps XMCO severity to Cortex XSOAR severity.
+    Converts the XMCO alert severity level ('none', 'low', 'medium', 'high', 'critical') to Cortex XSOAR alert
+    severity (1 to 4).
+
+    Args:
+        severity (str): severity as returned from the LePortail API.
+
+    Returns:
+        int: Cortex XSOAR Severity (0,5 to 4)
+    """
+    return {
+        'none': IncidentSeverity.INFO,
+        'low': IncidentSeverity.LOW,
+        'medium': IncidentSeverity.MEDIUM,
+        'high': IncidentSeverity.HIGH,
+        'critical': IncidentSeverity.CRITICAL
+    }[severity]
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'User <username> is authenticated' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'User <first_name> <surname> is authenticated' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        result = client.fetch_current_user()
+        if result.get('_error', None):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            message = f'User {result["user"]["first_name"]} {result["user"]["surname"]} is authenticated'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    demisto.results(message)
```

Comment:
The test result is returned in `main()`, it should not be returned twice.

In order to convey a successful test to the user, the string 'ok' should be returned. Any other result will fail the test. You can find out more about the `test-module` command [here](https://xsoar.pan.dev/docs/integrations/code-conventions#test-module).